README.md

```diff
@@ -11,94 +8,50 @@ Build status
 Introduction
 ------------
 
-This is a set of projects (bindings, code generator, examples and more) that enables developers to write ROS2
-applications in Rust.
+This is a set of projects (the `rclrs` client library, code generator, examples and more) that
+enables developers to write ROS 2 applications in Rust.
 
-Features
---------
+Features and limitations
+------------------------
 
 The current set of features include:
-- Generation of all builtin ROS types
+- Message generation
 - Support for publishers and subscriptions
 - Tunable QoS settings
 
-What's missing?
----------------
-
-Lots of things!
-- Component nodes
-- Clients and services
-- Tests
-- Documentation
+Lots of things are still missing however, see the [issue list](https://github.com/ros2-rust/ros2_rust/issues) for an overview.
 
-### Limitations
-
-- The `rclrs` interface is very limited for now and might not be idiomatic yet, any help and suggestion on the interface would be greatly appreciated
-- Due to the current ROS2 support of non-default clients, packages containing definitions of messages used in Rust crates must be present in the current workspace; otherwise message crates generation won't be triggered
+The client library is still rapidly evolving, and there are no stability guarantees.
 
 Sounds great, how can I try this out?
 -------------------------------------
 
-Here, the Foxy distribution of ROS 2 is used, but newer distributions can be used by simply replacing 'foxy' with the distribution name.
-
-```
-# First, make sure to have ROS 2 and vcstool installed (alternatively, install vcstool with pip):
-# sudo apt install ros-foxy-desktop ros-foxy-test-interface-files python3-vcstool libclang-dev clang
-# Install the colcon-cargo and colcon-ros-cargo plugins
-pip install git+https://github.com/colcon/colcon-cargo.git git+https://github.com/colcon/colcon-ros-cargo.git
-# Install the cargo-ament-build plugin
-cargo install cargo-ament-build
+In a nutshell, the steps to get started are:
 
-# In your workspace directory (ideally an empty one), run 
-mkdir src
+```shell
+mkdir -p workspace/src && cd workspace
 git clone https://github.com/ros2-rust/ros2_rust.git src/ros2_rust
+docker build -t ros2_rust_dev - < src/ros2_rust/Dockerfile
```

Comment:
I don't agree with recommending using Docker for developing `ros2_rust`. I use it myself, but the preferred way should be more in line with the rest of ROS2, which does not require Docker at all. I'm ok with adding a section about how to use Docker with `ros2_rust`, but should be in addition to explaining a Docker-less workflow.

Reply:
Sure. Do you mean that the method in the README should be the non-Docker way, or only that I should explain the non-Docker way in `Building.md`? I assumed that most people looking at the README would be interested in a quick and non-intrusive way of trying it out, hence I used the Docker setup.

Reply:
I think recommending Docker as the way to develop `ros2-rust` packages or the library itself is more intrusive, as it requires setting up a piece of software that is not strictly needed for ROS2 development. If someone has set up a ROS2 workspace already, they already have everything they need, and don't require Docker.

So, what I suggest is that we keep the Docker-less workflow, and then add an optional chapter about how to use Docker with `ros2-rust` for those of us who prefer to use it.

Reply:
@esteve Done.